docs/7.x/testplan.md

```diff
@@ -429,17 +431,39 @@ root$ gravity resource create cloud-config.yaml --confirm -m
 
 This scenario tests garbage collection on a cluster.
 
-Prerequisites: multi-node cluster with at least 1 regular node. Regular node is necessary to test both master and regular node update paths.
+Prerequisites: multi-node cluster with at least 1 node `--role=knode` and 1 `--role=node|master`.
+Having both `knode` and `node|master` is necessary to test both master and regular node update paths.
 
-Install a previous LTS version, upgrade to the latest version.
+Install a previous LTS version. Gather baseline pre-upgrade state:
 
-After upgrade execute `gravity gc` on the cluster.
+- [ ] `sudo du -sh /var/lib/gravity/{local/packages,site/packages,planet/registry}`
 
-- [ ] Verify the operation completes successfully.
- - [ ] Verify that packages from the previous installation have been removed locally.
- - [ ] Verify that packages from the previous installation have been removed from cluster package storage.
- - [ ] Verify that packages from the current installation are still present.
- - [ ] Tentative: Verify that application packages from remote clusters are still present.
+Upgrade to the release under test. Gather pre-garbage-collection state.
+
+- [ ] `sudo du -sh /var/lib/gravity/{local/packages,site/packages,planet/registry}`
+- [ ] List all images/tags in the registry
+ - `sudo find /var/lib/gravity/planet/registry/ -path '*tags/*' -type d | egrep 'tags/[^/]+$' | sort`
+- [ ] `sudo gravity system gc package --dry-run`
+
+Verify journal logs have been pruned [gravity.e#3429](https://github.com/gravitational/gravity.e/issues/3429)
+ - [ ] `sudo gravity exec du -h /var/log/journal` should show only one subdir.
+
+Execute `gravity gc`. Gather the operation id.
+
+- [ ] Verify the gc operation completes successfully.
+ - `sudo gravity plan --operation-id=<id>`
```

Comment:
nit: this could be without the operation ID as it will fallback to the last (attempted) operation, otherwise ID needs to be looked up either from `gravity status` or `gravity resource get operations`

Reply:
Fixed in 6ed5c45.

Now reads:

 - `sudo gravity plan` (`--operation-id` may be looked up from `gravity status` or `gravity resource get operations`)